pkg/virt-handler/isolation/isolation.go

```diff
@@ -59,6 +64,10 @@ type IsolationResult interface {
 	Mounts(mount.FilterFunc) ([]*mount.Info, error)
 	// returns the QEMU process
 	GetQEMUProcess() (ps.Process, error)
+	// returns the KVM PIT pid
+	KvmPitPid(vmpid int, vmi *v1.VirtualMachineInstance) (int, error)
```

Comment:
`vmpid` seems to be not required as argument as you can just call `GetQEMUProcess` inside `KvmPitPid`

Reply:
What I meant is that the struct implementing this interface also needs to have `GetQEMUProcess` method. Meaning you can just call it within `KvmPitPId`

```
func (r *RealIsolationResult) KvmPitPid() (int, error) {
        vmpid, err := r.GetQEMUProcess()
        if err != nil {
           return err
         }
	processes, _ := ps.Processes()
	nspid, err := GetNspid(vmpid)
	if err != nil || nspid == -1 {
		return -1, err
	}
	pitstr := "kvm-pit/" + strconv.Itoa(nspid)

	for _, process := range processes {
		if process.Executable() == pitstr {
			return process.Pid(), nil
		}
	}
	return -1, nil
```